lib/Inspection/include/Inspection/LoadInspectorBase.h

```diff
@@ -413,6 +421,21 @@ struct LoadInspectorBase : InspectorBase<Derived, Context> {
         });
   }
 
+  template<class T>
+  Status processSet(T& set) {
+    std::size_t idx = 0;
+    return this->self().doProcessList([&](auto value) -> Status {
+      auto ff = this->self().make(value);
+      typename T::value_type val;
+      if (auto res = process(ff, val); !res.ok()) {
+        return {std::move(res), std::to_string(idx), Status::ArrayTag{}};
+      }
+      set.insert(std::move(val));
```

Comment:
I couldn't find a neater way to process a set, beacuse  the function we found with `IsSetLike` has to be used after all.

Reply:
I think we can ditch this whole function and handle everything in `processList` like this:
```
  template<class T>
  Status processList(T& list) {
    constexpr isSet = detail::IsSetLike<T>::value;
    std::size_t idx = 0;
    return this->self().doProcessList([&](auto value) -> Status {
      auto ff = this->self().make(value);
      typename T::value_type val;
      if (auto res = process(ff, val); !res.ok()) {
        return {std::move(res), std::to_string(idx), Status::ArrayTag{}};
      }
      if constexpr (isSet) {
        list.insert(std::move(val));
      } else {
        list.push_back(std::move(val));
      }
      ++idx;
      return {};
    });
  }
  ```